boards/arm/nrf9160dk_nrf9160/CMakeLists.txt

```diff
@@ -50,6 +50,13 @@ if (CONFIG_BUILD_WITH_TFM)
 			 $<TARGET_PROPERTY:tfm,TFM_S_BIN_FILE>
 			 ${CMAKE_BINARY_DIR}/tfm_s_signed.bin
 
+		# Generate an intel hex file from the signed output binary
```

Comment:
Same comment as for nrf53, try doing this in separate statements to avoid merge-conflicts.